index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+  
+</head>
+<body>
+    <h1 id= "title">freeCodeCamp Survey Form</h1>
+    <p id ="description">Thank you for taking the time to help us improve the platform</p>
```

Comment:
i think this part is very stuck to the left side of the screen, it would be great if you add some padding-left.
so your project will lock more professional.

Reply:
Great job phrasing your comments as a suggestion.